test/runtests.jl

```diff
@@ -14,9 +14,10 @@ ENV["GKSwstype"] = "100"
 # environment settings
 islinux = Sys.islinux()
 istravis = "TRAVIS" ∈ keys(ENV)
+isappveyor = "APPVEYOR" ∈ keys(ENV)
```

Comment:
I think we should create a third variable `CI = istravis || isappveyor`, and then use it in all places where I was using `istravis`.